### PR TITLE
Michelle updated the statistics lesson to be the "modeling" lesson wi…

### PR DESCRIPTION
**Pull request by mvantieghem**
…th more higher-order stuff on syntax that can be applied across different models/packages. Can you please review the "Modeling_lesson.Rmd" file and provide feedback? then I will remove the out-dated versions. I don't think that the data challenge needs to change.

### CODE REVIEW

File: 5_models/Modeling_Lesson.Rmd [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+## Syntax: 
+### Basic model formula
+For any type of model with an IV and DV, there is a common syntax. The "~" delineates the direction of the model, where the DV is always on the left of the ~ and the predictors are always on the right.
+
+Y ~ X
+
+Example of t-test
+```{r}
+# ttest looking at the effect of ADHD group (IV) on dprime (DV)
+t.test(df$dprime ~ df$adhdF)
```
Comment (monicathieu) [inline 5_models/Modeling_Lesson.Rmd]: I think we might want to teach only the second method you have here (using the data argument) as it's a little cleaner and encourages people to only t-test columns in the same df. We could probably just remove this part and only include the instructions for the second one

File: 5_models/Modeling_Lesson.Rmd [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+summary(lm1)
+```
+
+We can index specific components of the output. What if I want to save the coefficients to a table?
+```{r}
+summary(lm1)$coefficients
+
+# save the table for later
+my_table <- summary(lm1)$coefficients
+save(my_table, file = "my_table.Rda")
```
Comment (monicathieu) [inline 5_models/Modeling_Lesson.Rmd]: I think, if we're going to teach people about saving model output to file, we should have them save at least the whole model object (which contains the model formula) for improved reproducibility, and to very carefully annotate `save()` commands in their scripts, so they don't run into version confusion where they don't know which data went into a certain model output.

File: 5_models/Modeling_Lesson.Rmd [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+There are many ways (and packages) to plot the output of your models. 
+We will start with a handy package that plots the effects of many model types.
+
+Effect Displays for Linear, Generalized Linear, and Other Models
+#https://cran.r-project.org/web/packages/effects/effects.pdf
+
+```{r}
+#install.packages("effects")
+library(effects)
```
Comment (monicathieu) [inline 5_models/Modeling_Lesson.Rmd]: this is 100% me being picky, feel free to ignore: I like to put all `library()` calls in the very first chunk of an Rmd so I can see them all in the same place. Given that the order in which packages are loaded affects which version of a function is called when multiple packages with a common function name are loaded (`dplyr::recode()` and `car::recode()`, for example), I try to put all package attaching at the top so it's really obvious what packages are loaded for a document and in what order.

File: 5_models/Modeling_Lesson.Rmd [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+summary(a2)
+```
+
+## Visualizing models 
+
+There are many ways (and packages) to plot the output of your models. 
+We will start with a handy package that plots the effects of many model types.
+
+Effect Displays for Linear, Generalized Linear, and Other Models
+#https://cran.r-project.org/web/packages/effects/effects.pdf
```
Comment (monicathieu) [inline 5_models/Modeling_Lesson.Rmd]: we can change this to a markdown-formatted hyperlink? I think this used to be a comment inside a code chunk, but it looks like it's outside a code chunk now so the single # will cause this to render as a level 1 header as is.

File: 5_models/Modeling_Lesson.Rmd (hunk 1 above)
Comment (pab2163) [reply 5_models/Modeling_Lesson.Rmd]: agreed with monica! I'll update this

**Comment by pab2163** [comment]
I made some updates to add the interpreting regression params part of this lesson -- it's a bit wordy so feel free to edit how you like!
